Source/WebCore/platform/network/DataURLDecoder.cpp

```diff
@@ -69,7 +69,7 @@ static WorkQueue& decodeQueue()
 static Result parseMediaType(const String& mediaType)
 {
     if (std::optional<ParsedContentType> parsedContentType = ParsedContentType::create(mediaType))
-        return { parsedContentType->mimeType(), parsedContentType->charset(), parsedContentType->serialize(), { } };
+        return { parsedContentType->mimeType().isolatedCopy(), parsedContentType->charset().isolatedCopy(), parsedContentType->serialize().isolatedCopy(), { } };
```

Comment:
I think this change is not related to this issue. Please add a line in the commit message explaining this change is needed.

Reply:
Oh oops, this is my other change.